connector/src/main/scala/com/basho/riak/spark/query/TSDataQueryingIterator.scala

```diff
@@ -0,0 +1,65 @@
+package com.basho.riak.spark.query
+
+import com.basho.riak.client.core.query.{ Location, RiakObject }
+import org.apache.spark.Logging
+import com.basho.riak.spark.util.DataConvertingIterator
+import com.basho.riak.spark.util.TSConversionUtil
+import org.apache.spark.sql.types.StructType
+import com.basho.riak.spark.rdd.TsTimestampBindingType
+import com.basho.riak.client.core.query.timeseries.ColumnDescription
+import scala.reflect.ClassTag
+
+class TSDataQueryingIterator[R](query: QueryTS)(implicit schema: Option[StructType] = None, tsTimestampBinding: TsTimestampBindingType, ct: ClassTag[R]) extends Iterator[R] with Logging {
+
+  private var _iterator: Option[Iterator[R]] = None
+  private val subqueries = query.queryData.iterator
+  println(query.queryData.size)
```

Comment:
Lets remove this debug print, or replace it with proper logging if it makes sense